server/frontend/src/components/Bugs/PossibleDuplicates/List.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div>
+    <button
```

Comment:
Please add the input text to manage summary instead of the django form (using a bootstrap group input for display)